<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsMenuWidget</class>
 <widget class="QWidget" name="SettingsMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="printerSettings">
     <property name="title">
      <string>Printer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="printerContent">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QComboBox" name="printerCombo">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editPrinterButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filamentSettings">
     <property name="title">
      <string>Filament</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="filamentContent">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QComboBox" name="filamentSelector"/>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="1">
            <widget class="QSpinBox" name="nozzleTempSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="bedTempLabel">
             <property name="text">
              <string>Bed Temp:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="bedTempSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="nozzleTempLabel">
             <property name="text">
              <string>Nozzle Temp:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="processSettings">
     <property name="title">
      <string>Process</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="processContent">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
            <item>
             <widget class="QComboBox" name="processPresetsSelector"/>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="qualityTab">
            <attribute name="title">
             <string>Quality</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Layer height</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="layerHeightLabel">
                 <property name="text">
                  <string>Layer height: </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="layerHeightSpinBox"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="strengthTab">
            <attribute name="title">
             <string>Strength</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1,0,1">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Walls</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="wallLoopsLabel">
                 <property name="text">
                  <string>Wall loops:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="wallLoopsSpinBox"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Infill</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="infillDensityLabel">
                 <property name="text">
                  <string>Infill density:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="infillDensityDoubleSpinBox">
                 <property name="suffix">
                  <string>%</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="infillPatternLabel">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Infill pattern:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="infillPatternComboBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="supportTab">
            <attribute name="title">
             <string>Support</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Support</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="1">
                <widget class="QCheckBox" name="enableSupportCheckBox"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="enableSupportLabel">
                 <property name="text">
                  <string>Enable support</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="typeLabel">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Type:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="typeComboBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
